packages/react-components/src/components/table-output-component.tsx

```diff
@@ -408,8 +416,8 @@ export class TableOutputComponent extends AbstractOutputComponent<TableOutputPro
         });
 
         if (this.columnApi) {
-            const columns = this.columnApi.getAllColumns();
-            const timestampHeader = columns.find(column => column.getColDef().headerName === 'Timestamp ns');
+            const columns = this.columnApi.getColumns();
```

Comment:
This seems not to work. columns is an empty array. Because of that the timestampHeader can't found and hence this.timestampCol is never set. This cause that time synchronization between view are not working because no signal is sent due to undefined this.timetampCol.

Set a breakpoint in method onEventClick() and when stepping through the code that no signal is sent.

Reply:
Good catch! `getColumns` is empty because state is empty. Resolved by applying `flushSync` to a `setState` function earlier in the class.